bundle/manifests/skupper-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,295 @@
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - pods/exec
+          - secrets
+          - configmaps
+          - services
+          - serviceaccounts
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        serviceAccountName: skupper-service-controller
+      - rules:
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - statefulsets
+          - daemonsets
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - pods/exec
+          - secrets
+          - configmaps
+          - services
+          - serviceaccounts
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        serviceAccountName: skupper-router
+      deployments:
+      - name: skupper-site-controller
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              control-plane: controller-manager
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                control-plane: controller-manager
+            spec:
+              containers:
+              - command:
+                - /app/site-controller
+                image: quay.io/skupper/site-controller@sha256:6618deb1c2cd45965f4f4f9cb46bdc4e37340188acf36e619f952df3255c0571
+                name: manager
+                securityContext:
+                  allowPrivilegeEscalation: false
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+              securityContext:
+                runAsNonRoot: false
+              serviceAccountName: skupper-site-controller
+              terminationGracePeriodSeconds: 10
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - skupper
+  - service
+  - mesh
+  - van
+  links:
+  - name: Skupper Operator
+    url: https://github.com/skupperproject/skupper-operator
+  maintainers:
+  - email: customerservice@redhat.com
```

Comment:
This is probably not the right email for the upstream. Perhaps put in the skupper google group?

Reply:
I wasn't sure but it makes sense to me as well. I will update it.